tests/probes/rpmverifypackage/test_probes_rpmverifypackage.sh

```diff
@@ -79,7 +81,9 @@ function test_probes_rpmverifypackage_noepoch {
 
 test_init
 
+if [[ $ID_LIKE != *"suse"* ]]; then
```

Comment:
Can we rather use `$ID` here? On my Fedora 29, there is no entry with `$ID_LIKE` in the `/etc/os-release` file. Also on RHEL-7.6 this is how the file looks like:
```
NAME="Red Hat Enterprise Linux Server"
VERSION="7.6 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
...
```

Reply:
Would you like me to check that it is a redhat based distro? Meaning something along this line:
` if [[ $ID = *"rhel"* ]]; then`

I wanted to be conservative with my change originally. So that the unit test is only disabled it on suse based systems. But I have not problem with making the change, just want to be sure I understand you correctly.

One concern I would have with the change mentioned above is that the unit tests would be disabled on rhel7 with that change. My understanding is that rhel7 does not have a `/etc/os-release`, at least that was my interpretation of the CI results to my last change for #1258. I haven't found the time to investigate the issue further yet. Meaning I am nut a 100% sure.

Reply:
@matusmarhefka, @jan-cerny sorry for the slow response I am quite busy atm.

I am okay to make the general change, but I have to investigate if this would work for openSUSE and SLE. I know that openSUSE would be okay. Still have to look into SLE.

Reply:
change needs to look like this if `$ID` is used:
```
if [[ $ID != *"sle"* && $ID != *"suse"* ]]; then
```